### PR TITLE
Bootstrap update

### PR DESCRIPTION
**Pull request by tomatics**

### CODE REVIEW

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -1,44 +1,54 @@
...
+          <li><%= link_to 'Payments', payments_path %></li>
+          <li><%= link_to 'Code Reviews', course_code_reviews_path(current_student.course) %></li>
+          <li><%= link_to 'Attendance record', attendance_statistics_path %></li>
+          <li><%= link_to 'Profile', edit_student_registration_path %></li>
+          <li><%= link_to "Log out", destroy_student_session_path, data: { method: :delete }, class: 'log-in' %></li>
+        <% elsif current_student && current_student.courses.empty? %>
+          <li><%= link_to 'Payments', payments_path %></li>
+          <li><%= link_to 'Profile', edit_student_registration_path %></li>
+          <li><%= link_to "Log out", destroy_student_session_path, data: { method: :delete }, class: 'log-in' %></li>
+    <% elsif current_admin %>
```
Comment (tomatics) [inline app/views/layouts/_navbar.html.erb]: fixed

File: app/views/layouts/_navbar.html.erb [hunk 2]
```diff
@@ -47,29 +57,32 @@
...
-            <b class="caret"></b> Internships
-          </a>
-          <ul class="dropdown-menu">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Internships <span class="caret"></span></a>
+          <ul class="dropdown-menu" role="menu">
             <li><%= link_to "Internships", course_internships_path(current_admin.current_course)  %></li>
             <li><%= link_to "Companies", companies_path %> </li>
           </ul>
         </li>
+
```
Comment (tomatics) [inline app/views/layouts/_navbar.html.erb]: done

File: app/views/layouts/_navbar.html.erb [hunk 3]
```diff
@@ -47,29 +57,32 @@
...
+      </ul>
+
+      <ul class="nav navbar-nav navbar-right">
         <li><%= link_to "Log out", destroy_admin_session_path, data: { method: :delete }, class: 'log-in' %></li>
       <% else %>
         <li><%= link_to "Teacher log in", new_admin_session_path %></li>
         <li><%= link_to "Student log in", new_student_session_path, class: 'log-in' %></li>
       <% end %>
-    </ul>
+      </ul>
```
Comment (tomatics) [inline app/views/layouts/_navbar.html.erb]: checked and adjusted.

File: app/views/layouts/_navbar.html.erb [hunk 4]
```diff
@@ -1,45 +1,47 @@
...
+          <li><%= link_to 'Code Reviews', course_code_reviews_path(current_student.course) %></li>
+          <li><%= link_to 'Attendance record', attendance_statistics_path %></li>
+          <li><%= link_to 'Profile', edit_student_registration_path %></li>
+          <li><%= link_to "Log out", destroy_student_session_path, data: { method: :delete } %></li>
+        <% elsif current_student && current_student.courses.empty? %>
+          <li><%= link_to 'Payments', payments_path %></li>
+          <li><%= link_to 'Profile', edit_student_registration_path %></li>
+          <li><%= link_to "Log out", destroy_student_session_path, data: { method: :delete } %></li>
+        <% elsif current_admin %>
         <li class="dropdown">
```
Comment (tomatics) [inline app/views/layouts/_navbar.html.erb]: nest

File: app/views/layouts/_navbar.html.erb [hunk 5]
```diff
@@ -1,45 +1,47 @@
...
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown">
-            <b class="caret"></b>
+          <a href="#" id="current-course" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
             <%= current_admin.current_course.description %>
-            <%= link_to '(edit)', edit_course_path(current_admin.current_course), class: 'fixit' %>
```
Comment (tomatics) [inline app/views/layouts/_navbar.html.erb]: remove class

File: app/views/layouts/_navbar.html.erb [hunk 6]
```diff
@@ -48,28 +50,27 @@
...
+          <ul class="dropdown-menu" role="menu">
             <li><%= link_to "Internships", course_internships_path(current_admin.current_course)  %></li>
             <li><%= link_to "Companies", companies_path %> </li>
           </ul>
         </li>
-        <li><%= link_to "Log out", destroy_admin_session_path, data: { method: :delete }, class: 'log-in' %></li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Log out", destroy_admin_session_path, data: { method: :delete } %></li>
       <% else %>
```
Comment (tomatics) [inline app/views/layouts/_navbar.html.erb]: nest

File: app/views/internships/edit.html.erb [hunk 7]
```diff
@@ -1,3 +1,3 @@
-<h1> Edit internship </h1>
+<h1> Edit internship at <b><i><%= @internship.company.name %></i></b></h1>
```
Comment (michaelrkn) [inline app/views/internships/edit.html.erb]: style with css

**Pull request merged**